assets/queries/terraform/azure/mariadb_server_georedundant_backup_disabled/test/positive2.tf

```diff
@@ -0,0 +1,17 @@
+resource "azurerm_mariadb_server" "positive2" {
+  name                = "example-mariadb-server"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  administrator_login          = "mariadbadmin"
+  administrator_login_password = "H@Sh1CoR3!"
```

Comment:
cc: @joaoReigota1 @rafaela-soares one more sample for passwords and secrets query